exonum-java-binding/core/rust/integration_tests/tests/service_runtime.rs

```diff
@@ -124,8 +124,8 @@ fn create_service_for_unknown_artifact() {
 }
 
 // Creates a new instance of JavaServiceRuntime for same JVM.
-fn get_runtime() -> JavaServiceRuntime {
-    JavaServiceRuntime::create_with_jvm(VM.clone(), 0)
+fn get_runtime() -> JavaRuntimeFactory {
```

Comment:
Comment is not updated. Though the first question is what JavaRuntimeFactory is (see the discussion there).